stun_server/region/module.tf

```diff
@@ -0,0 +1,37 @@
+locals {
+  service_name = "stun-server"
+  launch_type  = "FARGATE"
+}
+
+data "tfe_outputs" "infrastructure" {
+  organization = "home_assistant"
+  workspace    = "infrastructure"
+}
+
+module "stun_server" {
+  source = "../../.modules/service"
+
+  service_name      = local.service_name
+  container_image   = "ghcr.io/home-assistant/stun-server"
+  container_version = var.image_tag
+  launch_type       = local.launch_type
+  region            = var.region
```

Comment:
use the current region data source instead

Reply:
Changed. Also, used the region data source in other places as well.